x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_view.tsx

```diff
@@ -0,0 +1,189 @@
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection?.name,
+            section: 'integrate',
+          })
+        ),
+      isSelected: section === 'integrate',
+    },
+    {
+      id: 'settings',
+      label: i18n.translate('xpack.enterpriseSearch.analytics.collectionsView.tabs.settingsName', {
+        defaultMessage: 'Settings',
+      }),
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection?.name,
+            section: 'settings',
+          })
+        ),
+      isSelected: section === 'settings',
+    },
+  ];
+
+  useEffect(() => {
+    fetchAnalyticsCollection(name);
+  }, []);
+
+  return (
+    <EnterpriseSearchAnalyticsPageTemplate
+      restrictWidth
+      isLoading={isLoading}
+      pageChrome={[...collectionViewBreadcrumbs]}
+      pageViewTelemetry="View Analytics Collection"
+      pageHeader={{
+        description: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collectionsView.pageDescription',
+          {
+            defaultMessage:
+              'Dashboards and tools for visualizing end-user behavior and measuring the performance of your search applications. Track trends over time, identify and investigate anomalies, and make optimizations.',
+          }
+        ),
+        pageTitle: analyticsCollection?.name,
+        tabs: [...collectionViewTabs],
+      }}
+    >
+      {!analyticsCollection && (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiTitle>
+              <h2>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.analytics.collections.collectionsView.headingTitle',
+                  {
+                    defaultMessage: 'Collections',
+                  }
+                )}
+              </h2>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButtonTo fill iconType="plusInCircle" to={COLLECTION_CREATION_PATH}>
+              {i18n.translate(
+                'xpack.enterpriseSearch.analytics.collections.collectionsView.create.buttonTitle',
+                {
+                  defaultMessage: 'Create new collection',
+                }
+              )}
+            </EuiButtonTo>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      )}
+
+      <EuiSpacer size="l" />
+      {analyticsCollection ? (
+        <>
+          {section === 'settings' && (
+            <AnalyticsCollectionSettings collection={analyticsCollection} />
+          )}
+          {section === 'integrate' && (
+            <AnalyticsCollectionIntegrate collection={analyticsCollection} />
+          )}
+        </>
+      ) : (
+        <EuiEmptyPrompt
+          iconType="search"
+          title={
+            <h2>
+              {i18n.translate(
+                'xpack.enterpriseSearch.analytics.collections.collectionsView.emptyState.headingTitle',
```

Comment:
nitpick: better name for the message bundle id
`xpack.enterpriseSearch.analytics.collections.collectionsView.collectionNotFoundState.headingTitle`